misc/config_tools/upgrade_scenario_xml

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+function usage() {
+	echo "Usage: $0 <scenario XML> [<version of the given scenario XML>]"
+	echo "Upgrade the given scenario XML to align with the current version of ACRN."
+	echo ""
+	echo "Example: $0 logical_partition.xml v2.3"
+	echo "    The output file is logical_partition-v2.3.xml"
```

Comment:
Does that mean that this new file with extension `-v2.3` is in fact the new file for v2.5?

Reply:
calling the updated XML file ``-v2.3`` is not intuitive.  Also, the code in the script seems to use a computed "latest_ver_path" for the extension, not the name given on the command line?

Reply:
Yes, the generated file will have the latest version as the postfix. Will fix in v2.